source/datasets.rst

```diff
@@ -197,12 +197,14 @@ data by using a browser:
 
 #. Access the :strong:`Swagger interface` of the datasets in the
    Mobility domain, located at
-   https://mobility.api.opendatahub.bz.it/v2/swagger-ui.html. Like in
-   the case of the tourism' Swagger interface, you can learn REST API
-   call for that domain and fetch data for your application. There is
-   a dedicated howto to learn more how to interact with this
-   interface: ref:`get-started-mobility`
-
+   https://swagger.opendatahub.bz.it/?url=https://mobility.api.opendatahub.bz.it/v2/apispec. Like
```

Comment:
Better use https://mobility.api.opendatahub.bz.it, which makes the redirect then. Not point directly to the redirected swagger instance. I think it is more readable